### ISSUE TITLE
White flickering when opening NeoVim and setting colorscheme

### ISSUE THREAD

**Issue by sindriava** [rogual/neovim-dot-app#157]
When launching NeoVim, my window flickers in white for a second before I see the UI. This also seems to happen when I source $MYVIMRC.

This is apparently caused by setting the colorscheme, since commenting that line out stops the flicker. However, issuing that command manually doesn't cause the flicker to occur.

**Comment by sindriava** [comment]
I'm using OSX El Capitan, and the colorscheme is `base16-eighties`. As it seems, using `desert` doesn't cause the issue.

**Comment by philplckthun** [comment]
I think it's caused by #132 which is a quick fix for #130. Could that be @bambu?

**Comment by bambu** [comment]
Yup. Also #91 can be thrown into the mix. as well as a couple other issues in the neovim repo (referenced in 91)

The issue is that there is no way to launch nvim with flags that tell it to set the `gui_running` flag to true. So when the gui connects to nvim, it re-sources $MYVIMRC and as such you get the graphical flicker if you are going from one colorscheme to another or if the colorscheme you are using has special modes for when `gui_running` is set.

**Comment by rogual** [comment]
Perhaps we could hide the window and show it once init is complete. Although then if any errors occur you're stuck with a hidden window.

**Comment by sindriava** [comment]
That sounds like a fix, not a solution. This should probably be propagated directly to neovim, so a way to set the flag for GUI wrappers is implemented.

**Comment by rogual** [comment]
There was an issue on Neovim — https://github.com/neovim/neovim/issues/2216 — but it doesn't sound like they're going to change this.

**Comment by rogual** [comment]
Opened https://github.com/neovim/neovim/issues/3646 with Neovim, let's see what they say.
